tests/specs/npm/workspace_auto_install_basic/b/main.out

```diff
@@ -0,0 +1,4 @@
+Download http://localhost:4260/@denotest/esm-basic
+Download http://localhost:4260/@denotest/esm-basic/1.0.0.tgz
+Initialize @denotest/esm-basic@1.0.0
```

Comment:
We need to get this to create symlinks in the node_modules folder for the packages in the workspace.